media/js/src/solutionStep.tsx

```diff
@@ -40,6 +44,20 @@ export const SolutionStep: React.FC<SolutionStepProps> = (
 
     };
 
+    const handleDeleteStep = (
+        evt: React.MouseEvent<HTMLButtonElement>): void => {
+        evt.preventDefault();
+        const data = JSON.parse(
+            window.localStorage.getItem(
+                'question-' + id)) as ExerciseData[];
+        data[0].stepList.pop();
```

Comment:
Is it possible for the previous step to fail? I'm wondering if you need some guard code here as you index into the array. (Possibly not!)

Reply:
I don't believe so

Reply:
👍